apps/publish/content/common.py

```diff
@@ -594,6 +594,12 @@ def _publish_associated_items(self, original, updates=None):
                     original[ASSOCIATIONS][associations_key]['operation'] = self.publish_type
                     continue
 
+                # Do not publish removed association items on correction.
+                if (updates.get(ASSOCIATIONS, None)
+                        and associations_key in updates[ASSOCIATIONS]
+                        and updates[ASSOCIATIONS][associations_key] is None):
+                    continue
```

Comment:
I think you can simplify it with:
```
if updates.get(ASSOCIATIONS, {}).get(associations_key) is None:
    continue
```

Reply:
Actually, association_key must be present, so check for updates[ASSOCIATIONS][association_key] is None only if association_key is present in updates.

Reply:
makes sense